src/test/java/com/wikia/webdriver/testcases/mercurytests/curatedcontenttests/NavigationTests.java

```diff
@@ -16,16 +16,19 @@
  */
 public class NavigationTests extends NewTestTemplate {
 
+  private static final String ROOT_PATH = "/";
+  private static final String ROOT_PATH_SECTION = "/main/section/";
+  private static final String ROOT_PATH_CATEGORY = "/main/category/";
```

Comment:
I agree with Bogna that these paths could be reused in other tests
Test is not the best place for storing validated arguments
I'll prepare class for static curated content paths